pkg/kubernetes/events/events_test.go

```diff
@@ -31,7 +33,7 @@ func setup() {
 
 	eventRecorder := GenericEventRecorder()
 	if err := eventRecorder.Initialize(pod, kubeClient, "test"); err != nil {
-		log.Fatal().Err(err).Msg("Error initializing event recorder")
+		GinkgoT().Fatal("Error initializing event recorder")
 	}
```

Comment:
should we be using Ginkgo here considering this test is written using the standard Go testing library?

Reply:
I'd use https://golang.org/pkg/testing/#T.Fatal instead.

Reply:
@nojnhuh @shashankram No problem, I can close this PR off then. If that's the case then I presume [this issue](https://github.com/openservicemesh/osm/issues/1707) can be closed?